ledger/participant-integration-api/src/main/scala/platform/store/interfaces/TransactionLogUpdate.scala

```diff
@@ -99,11 +98,11 @@ object TransactionLogUpdate {
       commandId: String,
       workflowId: String,
       contractKey: Option[LfValue.VersionedValue],
-      treeEventWitnesses: Set[String],
-      flatEventWitnesses: Set[String],
-      submitters: Set[String],
+      treeEventWitnesses: Set[Party],
```

Comment:
I added my own GH comment at the inception of this PR motivating this.

Reply:
Not sure what you are referring to. On any rate my comment is general in that sense: if it is not possible to extract it is fine, if it would be possible, next time I would consider to extract that. Quality of review can be improved if meaningful changes do not collide with pure refactorings.